src/network/networkd-sysctl.c

```diff
@@ -179,6 +179,21 @@ static int link_set_ipv6_hop_limit(Link *link) {
         return sysctl_write_ip_property_int(AF_INET6, link->ifname, "hop_limit", link->network->ipv6_hop_limit);
 }
 
+static int link_set_ipv6_retransmission_time(Link *link) {
+        uint32_t retrans_time_ms;
+
+        assert(link);
+
+        if (!link_is_configured_for_family(link, AF_INET6))
+                return 0;
+
+        if (!timestamp_is_set(link->network->ipv6_retransmission_time))
+                return 0;
+
+        retrans_time_ms = (uint32_t) DIV_ROUND_UP(link->network->ipv6_retransmission_time, USEC_PER_MSEC);
```

Comment:
Also here. Please check the range of `DIV_ROUND_UP(...)` first.

Reply:
Done.